includes/mailpoet/class-repository.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * File containing the class Sensei_MailPoet_Repository.
+ *
+ * @since $$next-version$$
+ * @package sensei
+ */
+
+namespace Sensei\Emails\MailPoet;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Some Helper methods to fetch Sensei data: courses, groups and students.
+ */
+class Repository {
+	/**
+	 * Get all enrolled students in a course or group.
+	 *
+	 * @since $$next-version$$
+	 * @param int    $id The post ID.
+	 * @param string $post_type The post type.
+	 *
+	 * @return array
+	 */
+	public static function get_students( $id, $post_type ) {
```

Comment:
That's the matter of preference, probably... 
As for me, `static` methods are less preferable as it's impossible to mock them in tests. 
And in case the class has only static methods, it looks like the class itself is useless, it's just a set of namespaced functions.

Reply:
> Do you think we should make the methods instance methods?

I don't insist as I don't have unquestionable arguments :) And even across our small teams we have different views on this problem. And we don't have any guidelines for that. So, I leave it for you :)

But if you ask me, my personal opinion is that having an object with its instance methods, it is easier to tests code that uses those methods as you can mock the object's methods, but not static methods.

>  I just created it so the other classes won't be bloated with logic.

My opinion is to have more smaller classes instead of a few large classes. So I almost always consider it as a good idea to create a new class.